### ISSUE TITLE
mixer with faders

### ISSUE THREAD

**Issue by joopvdl** [cfoulc/cf#29]
I think your mixer is one of the best. But I have one request: could you make modules with faders, too? That would be awesome :)

**Comment by nay-seven** [comment]
I must admit, it's tempting , ;-) 
+1

**Comment by joopvdl** [comment]
nice long faders... i know you want them... hehehe :)

**Comment by nay-seven** [comment]
but I think it's not easy in terms of space ( the solo buttons, on ..+ the top connections) 
but who knows

**Comment by joopvdl** [comment]
Yes, it is a little crowded already. But of course you will gain the space that is occupied by the gain button. Maybe you can move the on, solo and pan buttons to the left, the fader could be in the middle.
Just thinking :)
Oh, one other suggestion, not really in the line of the philosophy of your mixer, but I would really use it: 4 ch or 8ch blocks, that can be connected just as the single channel blocks.  Would making setting up the mixer much faster and easier.
